WebRoot/js/zimbraMail/mail/controller/ZmComposeController.js

```diff
@@ -1228,7 +1228,7 @@ function() {
 		buttons.push(ZmOperation.SEND);
 	}
 
-	buttons.push(ZmOperation.CANCEL, ZmOperation.SEP, ZmOperation.SAVE_DRAFT);
+	buttons.push(ZmOperation.SAVE_DRAFT, ZmOperation.SEP, ZmOperation.CANCEL);
```

Comment:
Its good if we can get rid of ZmOperation.SEP from this toolbar. Though, I see on UI it has also been suppressed so not much effect from visibility perspective but its always good to keep DOM clean.